appcues/src/main/java/com/appcues/Appcues.kt

```diff
@@ -122,7 +121,12 @@ public class Appcues internal constructor(koinScope: Scope) {
      * @param properties Optional properties that provide additional context about the user.
      */
     public fun identify(userId: String, properties: Map<String, Any>? = null) {
-        identify(false, userId, properties)
+        if (userId.isEmpty()) {
```

Comment:
not sure if I could validate user id further down the line,  the return with log here could be placed somewhere else.